src/mmw/js/src/core/utils.js

```diff
@@ -254,6 +266,12 @@ var utils = {
 
             return l < r ? 1 : l > r ? -1 : 0;
         };
+    },
+
+    totalForPointSourceCollection: function(collection, key) {
+        return _.reduce(collection, function(accumulator, element) {
```

Comment:
Switch to `lodash`. It's already included as a dependency, and has [`_.sum`](https://lodash.com/docs#sum)